themes/app/src/scss/_typography.scss

```diff
@@ -77,7 +77,7 @@ h1 {
     font-weight: 700;
     font-size: 32px;
     color: $dark-grey;
-    line-height: 1.2;
+    line-height: 1.1;
```

Comment:
For the heading styles, you can actually refactor these to have all identical attributes under thhe same class grouping i.e. `h1, h2, h3...` have the identical attributes of colors, margins and font-familys

Then each heading class should only have the attributes that make them unique i.e. `font-sizes`, `font-weights` and `line-heights` 

Does that make sense?

Reply:
I would also move the mobile breakpoint changes for headings from the bottom to further up where the headings are in the file

Reply:
Will do 👌🏻